generators/python-bots/templates/python/bots/elements_form/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sym-api-client-python>=0.1.18
+sym-api-client-python>=1.3.6
```

Comment:
ohhh that was an old version, have you tried to generate it and checked the bot is running?